libmbp/devices/lg.cpp

```diff
@@ -91,58 +91,94 @@ void addLgOptimusGSeriesPhones(std::vector<Device *> *devices)
     devices->push_back(device);
 }
 
-void addLgOptimusLSeriesPhones(std::vector<Device *> *devices)
+static void CommonLgOptimusLSeries7x27a(std::vector<Device *> *devices)
```

Comment:
You want to pass the device variable you're modifying, not the entire list of devices.